client-side/src/stories/genericInput.scss

```diff
@@ -0,0 +1,78 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap');
+
+$color-border: rgb(61, 192, 155);
+$color-border-hover: rgb(45, 158, 126);
+$color-error: red;
+$color-black: black;
```

Comment:
please take the variables from stylies.scss file that includes all the variables,